libfwupd/fwupd-bios-attr.c

```diff
@@ -326,6 +326,94 @@ fwupd_bios_attr_set_description(FwupdBiosAttr *self, const gchar *description)
 	priv->description = g_strdup(description);
 }
 
+/* determine if key is supposed to be positive */
+static gboolean
+fu_bios_attr_key_is_positive(const gchar *key)
+{
+	if (g_strrstr(key, "enable"))
+		return TRUE;
+	if (g_strcmp0(key, "true") == 0)
+		return TRUE;
+	if (g_strcmp0(key, "1") == 0)
+		return TRUE;
+	if (g_strcmp0(key, "on") == 0)
+		return TRUE;
+	return FALSE;
+}
+
+/* determine if key is supposed to be negative */
+static gboolean
+fu_bios_attr_key_is_negative(const gchar *key)
+{
+	if (g_strrstr(key, "disable"))
+		return TRUE;
+	if (g_strcmp0(key, "false") == 0)
+		return TRUE;
+	if (g_strcmp0(key, "0") == 0)
+		return TRUE;
+	if (g_strcmp0(key, "off") == 0)
+		return TRUE;
+	return FALSE;
+}
+
+/**
+ * fwupd_bios_attr_map_possible_value:
+ * @self: a #FwupdBiosAttr
+ * @key: the string to try to map
+ * @error: (nullable): optional return location for an error
+ *
```

Comment:
i think you need some body here, i.e. what is the method *for* and why would you use it?